common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -862,11 +864,14 @@ protected AccountRecord getCachedAccountRecord(
                     );
         }
 
+        // TO-DO can we remove the parameters.getOAuth2TokenCache() instanceof MsalOAuth2TokenCache condition?
```

Comment:
I think as we discussed this was likely there to avoid someone configuring an AdalOAuth2TokenCache.  I'd remove this comment or update it.  You updated the overview comment to discuss this aspect a bit.  I suggest either linking to this PR here or filing a separate work item and then linking to that work item.

Reply:
added a work item for this here f2bb751adde2d05496e3cea965a9068022059612